assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -38,17 +39,30 @@ import { MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
 import SettingsControls from './SettingsControls';
 import GA4SettingsControls from './GA4SettingsControls';
 import EntityOwnershipChangeNotice from '../../../../components/settings/EntityOwnershipChangeNotice';
+import SettingsNotice, {
+	TYPE_WARNING,
+} from '../../../../components/SettingsNotice';
+import Link from '../../../../components/Link';
 import { isValidAccountID } from '../../util';
 import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
+import { CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+import { useFeature } from '../../../../hooks/useFeature';
 const { useSelect } = Data;
 
 export default function SettingsForm( {
 	hasAnalyticsAccess,
 	hasAnalytics4Access,
 } ) {
+	const ga4Enabled = useFeature( 'ga4Reporting' );
```

Comment:
Please can you rename this for consistency with naming elsewhere in the codebase? It would also be a little clearer:

```suggestion
	const ga4ReportingEnabled = useFeature( 'ga4Reporting' );
```

Reply:
Updated.